public/index.html

```diff
@@ -26,6 +26,7 @@
     -->
 
     <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Unna&display=swap" rel="stylesheet">
```

Comment:
a couple things for performance improvements:

1. do we know which weights of fonts we need to use? there's no benefit to including font weights and styles that we know we're not going to us. this isn't a huge deal, but in my opinion it's always best to request the least amount of data needed

2. we can combine these two calls into one to reduce network latency/connection times. 

using both those techniques into something like 

```
<link href="https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&family=Unna:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
```